source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -26,31 +26,117 @@ content: |
 
      [MongoDBODBC]
      Description = ODBC for MongoDB BI Connector
-     Driver      = /usr/local/lib/<libmdbodbca.so|libmdbodbcw.so>
+     Driver      = /usr/local/lib/libmdbodbcw.so
      Trace       = Off
      TraceFile   = stderr
      ReadOnly    = yes
-     SERVER      = <server address>
-     PORT        = 3307
-     USER        = <username>
-     PASSWORD    = <password>
-     DATABASE    = <dbname>
+     Server      = <server address>
+     Port        = <port number>
+     User        = <username>
+     Password    = <password>
+     Database    = <dbname>
 
      ; optional SSL configuration
 
      SSLKEY      = </path/to/.pem-key-file>
      SSLCERT     = </path/to/ssl-certificate>
-     SSLMODE     = DISABLE|ALLOW|PREFER|REQUIRE
+     SSLMODE     = DISABLED|PREFERRED|REQUIRED
      SSLCA       = </path/to/ssl-ca-list-file>
      SSLCAPATH   = </path/to/ssl-ca-file-directory>
 
+  .. note::
+
+     All ``odbc.ini`` parameters are case-insensitive.
+
+  The following table describes the parameters used in the above DSN.
```

Comment:
Is the odbc.ini the DSN? If so, it may help to spell that out or the relationship between the DSN and the odbc.ini file.

Reply:
No, you can have multiple DSNs in one odbc.ini. Added a note to that effect.